src/components/filter/expert/expert-filter-utils.ts

```diff
@@ -407,6 +519,22 @@ export const queryValidator: QueryValidator = (query) => {
                 valid: false,
                 reasons: [RULES.EMPTY_RULE],
             };
+        } else if (rule.id && dataType === DataType.COMBINATOR) {
+            // based on FIELDS_OPTIONS configuration and group rule value, validate for each child rule in a group rule
+            const childFields = Object.keys(
```

Comment:
```suggestion
            const childrenFields = Object.keys(
```

Reply:
DONE in https://github.com/gridsuite/commons-ui/pull/452/commits/f8a71a4e4be081d578720514b66b295e3d65ef93